app/client/src/components/pages/Community/components/tabs/Overview.js

```diff
@@ -88,7 +88,6 @@ const toggleStyles = css`
 function Overview() {
   const {
     cipSummary,
-    assessmentUnitCount, // TODO: determine if this is needed...
```

Comment:
😄 thank you. I added that comment in my stream gages PR, and forgot to revisit it. It looks like the `LocationMap` component is the only other file where this is used, so I'm wondering if we can use `waterbodies.length` there too as well? That could be a small change as part of a separate PR.